chemprop/cli/train.py

```diff
@@ -489,22 +489,41 @@ def normalize_inputs(train_dset, val_dset, args):
     if d_xd > 0 and not args.no_descriptor_scaling:
         X_d_scaler = train_dset.normalize_inputs("X_d")
         val_dset.normalize_inputs("X_d", X_d_scaler)
-        logger.info(f"Features: loc = {X_d_scaler.mean_}, scale = {X_d_scaler.scale_}")
+        
+        X_d_scaler = [X_d_scaler] if not isinstance(X_d_scaler, list) else X_d_scaler
+
+        for i, scaler in enumerate(X_d_scaler):
+            logger.info(f"Features for mol {i}: loc = {scaler.mean_}, scale = {scaler.scale_}")
```

Comment:
There is only one unique X_d_scaler per datapoint even if it is a multicomponent datapoint. So this only needs to print once.